documentation/user-interface/README.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: docs
+title: User Interface
+permalink: /documentation/user-interface/
+---
+# User Interface
+
+* [Explorer Panel](explorer-panel.md)
+* [Record Configuration Dialog](record-configuration-dialog.md)
+* [Run Configurations Dialog](run-configurations-dialog.md)
```

Comment:
Probably from user's point of view I would expect find descriptions for all views (about reports and ActionPanel too, or at list links for this sections) in this section